tests/queries/0_stateless/03198_dynamic_read_subcolumns.sql

```diff
@@ -1,4 +1,4 @@
--- Tags: no-random-settings, no-s3-storage
+-- Tags: no-random-settings, no-object-storage
```

Comment:
Do you mind adding a comment below explaining why the test doesn't work with object storage please? Just in case we forget about it in the future

Reply:
Sure